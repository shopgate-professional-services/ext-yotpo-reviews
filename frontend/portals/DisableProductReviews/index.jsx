/**
 * @returns {null}
 */
const DisableComponent = () => null;

export default DisableComponent;
