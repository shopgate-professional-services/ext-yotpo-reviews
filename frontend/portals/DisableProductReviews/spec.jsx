import React from 'react';
import { shallow } from 'enzyme';
import DisableProductReviews from './index';

describe('<DisableProductReviews />', () => {
  it('should return null', () => {
    const wrapper = shallow((
      <DisableProductReviews />
    ));

    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('div').exists()).toBe(false);
  });
});
