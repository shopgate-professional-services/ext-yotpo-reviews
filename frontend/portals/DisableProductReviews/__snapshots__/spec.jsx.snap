// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DisableProductReviews /> should return null 1`] = `""`;
